app/routes/index.tsx

```diff
@@ -53,6 +54,18 @@ export default function App() {
     .filter(({questionState}) => questionState === '_')
     .map(({title}) => title)
 
+  const handleSpecialLinks = (e: MouseEvent) => {
+    const el = e.target as HTMLAnchorElement
+    if (el.tagName !== 'A') return
```

Comment:
is this always uppercase?

Reply:
yes, https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName